dictionaries/html/src/html.txt

```diff
@@ -1,24 +1,28 @@
+__utma
```

Comment:
These are google analytics cruft https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage?csw=1#HowGAUsesCookies

Reply:
Do you think they should be included?
I see them all over the place.

Reply:
Maybe `fullstack`? not really and HTML thing, but likely common enough that someone would appreciate it in a dictionary